primes.rb

```diff
@@ -1,6 +1,60 @@
 class Primes
-  def self.sum_to(limit = 100)
-    # TODO - add your prime number solution here...
-    "I'm working on it!"
-  end
+  
+	# I understand how to make this run with input from the user throught the terminal (limit = gets.chomp)
+	# But I couldn't succesfull get it to work with the .erb files
+
+	require 'prime'
+
+	def self.sum_to(limit = 100)
+		sum = 0
+		Prime.each(limit) do |prime|
```

Comment:
This works but also loses the opportunity to play around with some good prime number algorithms